lib/jefferies_tube/railtie.rb

```diff
@@ -13,9 +13,7 @@ class Railtie < ::Rails::Railtie
       ARGV.push "-r", File.join(File.dirname(__FILE__),"custom_prompts.irbrc.rb")
 
       if defined? Pry
-        Pry.config.prompt = proc {
-          JefferiesTube::Console.prompt
-        }
+        Pry.prompt = Pry::Prompt.new(:jefferies_tube, '', Array.new(2) { proc { JefferiesTube::Console.prompt } })
```

Comment:
I might need to add a minimum version of `pry` for this to work. 

I think it's v0.13.0, according to this changelog: https://github.com/pry/pry/blob/master/CHANGELOG.md#v0130-march-21-2020